pkg/docker/util.go

```diff
@@ -363,3 +363,16 @@ func GetDefaultDockerConfig() *api.DockerConfig {
 
 	return cfg
 }
+
+// GetAssembleUser finds an assemble user on the given image.
+func GetAssembleUser(cfgAssembleUser string, image string, client Client, config *api.Config) (string, error) {
```

Comment:
Update this function to use the config object to pull all of the information instead of passing two strings (cfgAssembleUser, image) that are contained in config separately.